pkg/code-generator/schemagen/crd.go

```diff
@@ -22,6 +22,8 @@ const (
 	v1 = "apiextensions.k8s.io/v1"
 )
 
+const crdFileHeader = `CRD ValidationSchema generated by solo-kit. DO NOT EDIT.`
```

Comment:
Is `ValidationSchema` meaningful? The only instance of that string I see in the `gloo` repo is the `ValidationSchemaOptions`. Can we change it or just make it "validation schema"?